github/resource_github_organization_role_test.go

```diff
@@ -12,6 +12,7 @@ func TestAccGithubOrganizationRole(t *testing.T) {
 	t.Run("can create an empty organization role", func(t *testing.T) {
 		randomID := acctest.RandStringFromCharSet(5, acctest.CharSetAlphaNum)
 		name := fmt.Sprintf("tf-acc-org-role-%s", randomID)
+		baseRole := "none"
```

Comment:
This test is testing the optional base role defaulting to none, so this change breaks the test.